src/background/modules/OmniboxSearch.js

```diff
@@ -201,6 +201,11 @@ export async function triggerOmnixboxSearch(text, disposition) {
  * @throws TypeError
  */
 async function toggleEnabledStatus(toEnable) {
+    // Thunderbird
+    if (typeof messenger !== "undefined") {
```

Comment:
Hmm why even bother about this here, as there is not even any Omnibox on Thunderbird?

Reply:
The `toggleEnabledStatus()` function is called twice below, so it would obviously cause errors in TB without this return statement.